agenthub/codeact_agent/prompt.py

```diff
@@ -39,6 +39,7 @@
 SYSTEM_SUFFIX = """Responses should be concise.
 The assistant should attempt fewer things at a time instead of putting too many commands OR too much code in one "execute" block.
 Include ONLY ONE <execute_ipython>, <execute_bash>, or <execute_browse> per response, unless the assistant is finished with the task or needs more input or action from the user in order to proceed.
+If the assistent is finished with the task you MUST include <finish></finish> in your response.
```

Comment:
```suggestion
If the assistant is finished with the task you MUST include <finish></finish> in your response.
```

Reply:
Unfortunately this means all prompts have to be patched again.

I would just do

```
ONLY_TEST_AGENT=CodeActAgent ./tests/integration/regenerate.sh
```

(without `SKIP_TEST=true FORCE_USE_LLM=true`)

which should regenerate the prompts and patch them.